src/ArborX.hpp

```diff
@@ -23,6 +23,7 @@
 #include <ArborX_LinearBVH.hpp>
 #include <ArborX_Point.hpp>
 #include <ArborX_Predicates.hpp>
+#include <ArborX_Ray.hpp>
```

Comment:
Please don't.  This was intentional.  You had to explicitly include it to get support for raytracing.
I am not necessarily opposed to revisit but it does not belong here.